### PR TITLE
Try adding header to generated CLI docs

### PR DESCRIPTION
**Pull request by justinsb**
This should stop me changing the wrong place!

Fixes: https://github.com/kubernetes/kops/issues/2834

### CODE REVIEW

File: cmd/kops/gen_help_docs.go [hunk 1]
```diff
@@ -21,6 +21,11 @@ import (
 	"github.com/spf13/cobra/doc"
 )
 
+const fileHeader = `
+<!--- This file is automatically generated by make gen-cli-docs; changes should be made in the go CLI command code -->
```
Comment (chrislovecnm) [inline cmd/kops/gen_help_docs.go]: > go CLI command code
Should we say where the code is?  cmd/pkg/?

There are a lot of first timers that are making these changes.  Make it simple for them.

**Comment by chrislovecnm** [comment]
I added the issue https://github.com/kubernetes/kops/issues/2834 that this will close.

**Comment by justinsb** [comment]
Done!

**Comment by chrislovecnm** [comment]
/assign
/lgtm

you can self merge if you beat me to it

**Pull request merged**